server/app/views/admin/programs/ProgramBlockPredicatesEditView.java

```diff
@@ -310,22 +314,22 @@ private ContainerTag createScalarDropdown(QuestionDefinition questionDefinition)
         .setLabelText("Field")
         .setCustomOptions(options)
         .addReferenceClass(ReferenceClasses.PREDICATE_SCALAR_SELECT)
-        .getContainer();
+        .getSelectTag();
   }
 
-  private ContainerTag createOperatorDropdown() {
-    ImmutableList<ContainerTag> operatorOptions =
+  private DivTag createOperatorDropdown() {
+    ImmutableList<OptionTag> operatorOptions =
         Arrays.stream(Operator.values())
             .map(
                 operator -> {
                   // Add this operator's allowed scalar types as data, so that we can determine
                   // whether to show or hide each operator based on the current type of scalar
                   // selected.
-                  ContainerTag option =
-                      option(operator.toDisplayString()).withValue(operator.name());
+                  OptionTag option = option(operator.toDisplayString()).withValue(operator.name());
                   operator
                       .getOperableTypes()
-                      .forEach(type -> option.attr("data-" + type.name().toLowerCase()));
+                      .forEach(type -> option.withData(type.name().toLowerCase(), ""));
+                  // to be passed?
```

Comment:
Can this now be removed?

Reply:
you mean just delete the `.forEach(..)` call?

Reply:
oh, the comment (facepalm lol)